### ISSUE TITLE
Different sequence runs shows significantly different in alpha diversity

### ISSUE THREAD

**Issue by Leran10** [benjjneb/dada2#1565]
Hi,

I have 40 samples split into 4 16s sequencing runs. But after merge the 4 runs and do alpha diversity analysis, I found they show significantly statistical difference from each other.

All these conditions should be very similar. The only differences are each batch may have different sequencing death. The total reads we got are different batch by batch.

So I wondered, if this is normal or an evidence that we have something went wrong during wet lab processing?

Thanks!
Leran

**Comment by benjjneb** [comment]
What alpha diversity metric are you analyzing here?

**Comment by Leran10** [comment]
Hi, We used Observed richness and Shannon diversity.

**Comment by benjjneb** [comment]
Observed taxa is highly dependent on sequencing depth. Shannon Index is much less so, but not entirely immune to it. Could you recreate these plots from a set of samples subsampled to a constant sequencing depth? (this goes by the term "rarefy" in microbiome analysis) That would be helpful as to trying to understand how systematic the differences are between "W"s.

That said, there are not large amounts of samples in each "W". Is it possible there is a real difference between the samples in each run?

**Comment by Leran10** [comment]
Thank! We used Rarefy() to subsample them to depth of 7000. And 8 samples and 97 OTUs were removed. The updated plot is as below:

![image](https://user-images.githubusercontent.com/17597071/174679746-dc1abe5d-7962-49ee-8002-e1664a92a3f6.png)

P values of pairwise Wilcoxon tests are not significant anymore, but the medians of observed ASVs of W2 and W4 are higher than W1 and W3.

So it seems that using this method we still cannot completely get rid of their differences by sequencing batch. 

And all the runs are normal cohort without any treatment.....
